### ISSUE TITLE
Font size not readable in visualisations 

### ISSUE THREAD

**Issue by janagombitova** [akvo/akvo-lumen#3057]
We received feedback recently that the text showing in Lumen visualisations is hard to read due to the size of the font used. This is especially for cases when the visualisations are used offline, but actually when being created and shared online some of the fonts are really small. 

The goal of the visualisations in Lumen is to clearly share an insight/a message, meaning the person viewing it needs to be able to also read what is written in the visualisations. This issue goes hand in hand with https://app.zenhub.com/workspaces/akvo-flow--lumen-55d493e09e846e9104ab9478/issues/akvo/akvo-lumen/2818

Secondly, when setting up visualisations we want Salim to have the flexibility to customise them to his needs but also limit the decisions he needs to take. We need to see what is the best way in this case.  

Here is an example of this visualisation (https://akvointernal.akvolumen.org/visualisation/5e9d5a3f-a139-4da6-b58b-74d7ce7a7a79) in the editor, shared online and downloaded as a png. 

![Screen Shot 2020-11-26 at 11 06 09](https://user-images.githubusercontent.com/12456965/100337486-ba007b80-2fd7-11eb-8a46-3bd802047d30.png)
*^ in the editor* 

![Screen Shot 2020-11-26 at 11 08 20](https://user-images.githubusercontent.com/12456965/100337583-d8667700-2fd7-11eb-9e0f-0133e0e72b87.png)
*^shared online - https://akvointernal.akvolumen.org/s/52nX-XVx8ps* 
![New vs Returning visits ](https://user-images.githubusercontent.com/12456965/100337574-d56b8680-2fd7-11eb-847e-a64f1006abc2.png)
*^ downloaded as a png*

@Kiarii what is your take?

**Comment by Kiarii** [comment]
This is an issue that Molo also shared yesterday 😟

For a quick turn around we would need to increase the font size, spacing and weight before we start looking into whether we can provide customizations around the fonts..

**Comment by janagombitova** [comment]
Let's make that quick turnaround happen. Can you define the specs @Kiarii ?

**Comment by janagombitova** [comment]
One issue that might be worth to look at either together with this issue or to keep in mind when thinking of the solution: https://github.com/akvo/akvo-lumen/issues/2818

**Comment by kardan** [comment]
Started to look at this issue yesterday. Some (legends) of the texts are easy to increase by changing css. Others like the texts in the AxisLeft is not respecting the calculated axisLabelFontSize (https://github.com/akvo/akvo-lumen/blob/master/client/src/components/charts/bar/SimpleBarChart.jsx#L273)
Investigating locally my local axisLabelFontSize is 13 but HTML generated is
```
<svg x="-0.25em" y="0.25em" font-size="10" style="overflow: visible;"><text x="-8" y="796.125" fill="#222" font-family="Arial" font-size="10" text-anchor="end"><tspan x="-8" dy="0em">0.5</tspan></text></svg>
```

**Comment by Kiarii** [comment]
I've defined the [preliminary changes relating only to text/font here](https://www.figma.com/file/XQtjy781ugZ8dmmaA9hWnO/Lumen-style-guide?node-id=1786%3A0), ignore the rest of the file which will focus on visual spacing and grouping elements and components

**Comment by kardan** [comment]
Font sizes for bar charts (normal, stacked, vert & horizontal) been increased. We should review how things looks on test were there are more different charts etc.

**Comment by janagombitova** [comment]
Here is how the example visualisation looks now: 

![Screen Shot 2021-01-06 at 13 32 23](https://user-images.githubusercontent.com/12456965/103768849-87f52700-5023-11eb-941e-670695d77881.png)

^ in the editor 

![Screen Shot 2021-01-06 at 13 33 20](https://user-images.githubusercontent.com/12456965/103768912-a22f0500-5023-11eb-895a-b24e91770dcb.png)

^ shared online 

![New vs Returning visits ](https://user-images.githubusercontent.com/12456965/103768944-b4a93e80-5023-11eb-8adf-a09409d0e30a.png)

^ png 

The components are more visible and better readable. Good work you two!

**Comment by janagombitova** [comment]
Can we also handle the following: 

Increase the axis fonts for line and area charts to match the current bar charts one 
![Screen Shot 2021-01-06 at 13 39 23](https://user-images.githubusercontent.com/12456965/103769424-8415d480-5024-11eb-9f73-27636191f7ae.png)

Increase the font for the bubble chart and scatter plot legend - Size - description to match the bucket list font size
![Screen Shot 2021-01-06 at 13 40 44](https://user-images.githubusercontent.com/12456965/103769516-b9babd80-5024-11eb-8815-2d8ae71b154d.png)

Match the visualisation title and update date with the bar charts for pivot tables (if not already done) 
![Screen Shot 2021-01-06 at 13 43 01](https://user-images.githubusercontent.com/12456965/103769707-128a5600-5025-11eb-9bf1-864d852008c8.png)
